tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -973,11 +1012,13 @@ private function loadFixture()
         $this->userId = $user1->getId();
         $this->userId2 = $user2->getId();
         $this->groupId = $group1->id;
+        $this->ddc2504OtherClassId = $otherClass->id;
+        $this->ddc2504ChildClassId = $childClass1->id;
 
+        $this->username = $user1->username;
+        $this->groupname = $group1->name;
         $this->topic = $article1->topic;
         $this->phonenumber = $phonenumber1->phonenumber;
-        $this->ddc2504OtherClassId = $otherClass->id;
```

Comment:
Not really... it's just that others were purely IDs... and at this block it was only objects, that's why I moved them together